lib/smux.h

```diff
@@ -143,6 +143,7 @@ extern int smux_trap_multi_index(struct variable *vp, size_t vp_len,
 				 size_t trapobjlen, uint8_t sptrap);
 extern int oid_compare(const oid *, int, const oid *, int);
 extern void oid2in_addr(oid[], int, struct in_addr *);
+extern void oid2in6_addr(oid[], struct in6_addr *);
```

Comment:
Can we fix the decleration here?  it's incorrect and does not follow our standards

`void oid2in6_addr(oid oid[], struct in6_addr *addr)`

Reply:
ack. done.